packages/web-twig/.dockerignore

```diff
@@ -0,0 +1,30 @@
+**/*.log
```

Comment:
I guess I'd plunge the whole thing into some kind of subfolder? Not combine bundle and symfony app for examples..

Reply:
Moved to a different directory.

Reply:
App skeleton was moved to another directory.